pkg/kev/config/extensions.go

```diff
@@ -148,6 +149,14 @@ func WorkloadRestartPolicyFromCompose(svc *composego.ServiceConfig) string {
 		policy = svc.Deploy.RestartPolicy.Condition
 	}
 
+	if policy == "unless-stopped" {
+		log.WarnWithFields(log.Fields{
+			"restart-policy": policy,
+		}, "Restart policy 'unless-stopped' is not supported, converting it to 'always'")
+
+		policy = "always"
```

Comment:
OK, this function now returns what has been defined in the docker compose around restart policy, however, we prime the value of policy in the first statement to be `config.RestartPolicyAlways` which is a Kev config parameter value. Perhaps change it to literal string "always" instead?